assets/pages/contributions-thankyou/oneoffContributionsThankyou.jsx

```diff
@@ -12,12 +12,19 @@ import SocialShare from 'components/socialShare/socialShare';
 
 import { statelessInit as pageInit } from 'helpers/page/page';
 import { renderPage } from 'helpers/render';
+import { detect as detectCountry } from 'helpers/internationalisation/country';
+import { detect as detectCurrency } from 'helpers/internationalisation/currency';
+import { trackComponentEvents } from './helpers/trackComponentEvents';
 
 
 // ----- Page Startup ----- //
 
 pageInit();
+const campaignCode = 'oneoff-thankyou-page-recurring-upsell';
```

Comment:
Would it make sense to call this something like `onwardCampaignCode` or `upSellCampaignCode`? There will already be something called `campaignCode` sitting in the sessionStorage on this page from the campaign that sent people here in the first place. Could be a possible source of confusion?